class.jetpack.php

```diff
@@ -577,6 +569,10 @@ private function __construct() {
 			Jetpack_Heartbeat::init();
 			require_once JETPACK__PLUGIN_DIR . '_inc/lib/class.jetpack-search-performance-logger.php';
 			Jetpack_Search_Performance_Logger::init();
+
+			// Add wordpress.com to the safe redirect whitelist if Jetpack is connected
+			// so the customizer can `return` to wordpress.com if invoked from there.
+			add_action( 'customize_register', array( $this, 'add_wpcom_to_allowed_redirect_hosts' ) );
```

Comment:
how is this realated to manage?

Reply:
I don't know but it was in the original manage module code.

https://github.com/Automattic/jetpack/blob/branch-5.9/modules/manage.php

Reply:
When administering a site via WordPress.com, a Jetpack site will have a "Customize" option that redirects to WP-Admin like:
https://example.com/wp-admin/customize.php?return=https%3A%2F%2Fwordpress.com%2Fstats%2Fday%2Fexample.com

On the Jetpack site, the "x" in the top-left of the Customizer will return that URL, but it is first passed through `wp_validate_redirect` which ensures the redirect is going to an allowed domain. This function will add `wordpress.com` to that list so the 'x' link will work.

It was part of Manage since that's how someone would have a Customize option in WP.com.